com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -911,8 +906,14 @@ public void Shutdown()
                 BehaviourUpdater = null;
             }
 
-            //The Transport is set during Init time, thus it is possible for the Transport to be null
-            NetworkConfig?.NetworkTransport?.Shutdown();
+            // NSS TODO: Remove this once MTT-860 is addressed or before PR
```

Comment:
MTT-860 Might not get addressed before this PR and I might just remove the NSS TODO portion but leave the reference to MTT-860.  The scenario is if a user reloads a scene (i.e. game over and restart a new game while still in the same session) that has the NetworkManager within it and the NetworkManager has Don't Destroy enabled then another NetworkManager instance is created causing a conflict between the existing NetworkManager in the DoNotDestroyOnLoad scene and the one being instantiated when the scene loads.  
This check prevents the newly instantiated NetworkManager from shutting down the Transport as that will kill the currently existing game session for the players and host.